tests/dependency_analyzer/cyclonedx/test_cyclonedx.py

```diff
@@ -106,3 +107,11 @@ def test_multiple_versions(snapshot: dict[str, DependencyInfo]) -> None:
     bom_path = Path(RESOURCES_DIR, "bom_multi_versions.json")
     result = get_deps_from_sbom(bom_path)
     assert snapshot == result
+
+
+def test_custom_sbom_name_with_maven() -> None:
+    """Test reading cyclonedx maven sbom that was created using a custom name."""
+    cyclonedx: CycloneDxMaven = CycloneDxMaven(
+        "", "bom.json", "maven", defaults.get("dependency.resolver", "dep_tool_maven"), "localhost"
```

Comment:
nit: is there any reason why the `repo_path` attribute of `CycloneDxMaven` is `localhost`?

Reply:
Not really. It's not used so perhaps an empty string is better.